zxlive/proof_panel.py

```diff
@@ -421,6 +421,10 @@ def _refresh_rewrites_model(self) -> None:
         # TODO: Right now this calls for every single vertex selected, even if we select many at the same time
         self.graph_scene.selectionChanged.connect(model.update_on_selection)
 
+    def shutdown(self):
```

Comment:
Why do you need this method here in `proof_panel` if it is already implemented in `base_panel`?